lib/node_modules/@stdlib/repl/lib/commands/set_keybinding.js

```diff
@@ -0,0 +1,68 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable no-underscore-dangle */
+
+'use strict';
+
+// MODULES //
+
+var format = require( '@stdlib/string/format' );
+var contains = require( '@stdlib/assert/contains' );
+var isString = require( '@stdlib/assert/is-string' );
+var ACTIONS = require( './../actions.js' );
+
+
+// MAIN //
+
+/**
+* Returns a callback to be invoked upon calling the `setKeybinding` command.
+*
+* @private
+* @param {REPL} repl - REPL instance
+* @returns {Function} callback
+*/
+function command( repl ) {
+	return onCommand;
+
+	/**
+	* Allows the user to set a keybinding.
+	*
+	* @private
+	* @param {string} action - action name
+	* @returns {void}
+	*/
+	function onCommand( action ) {
```

Comment:
@Snehil-Shah I think it would also be worthwhile to support an optional second argument which sets the keybinding directly. If not provided, then listen for keypresses.